packages/shared-bridge/src/dto/agrement.dto.ts

```diff
@@ -0,0 +1,96 @@
+import { ACTIVITE_TYPE, AGREMENT_STATUT } from "../constantes/agrement";
+import { FILE_CATEGORY } from "../constantes/file";
+
+export interface AgrementsDto {
```

Comment:
```suggestion
export interface AgrementDto {
```
le type ne représente que un seul agrement, sinon ça donne l'impression de manipuler un array :
```js
type AgrementsDto = Array<AgrementDto>
```